tck/src/main/java/org/eclipse/microprofile/lra/tck/participant/api/LRATypeTckBean.java

```diff
@@ -0,0 +1,93 @@
+/*
+ *******************************************************************************
+ * Copyright (c) 2020 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package org.eclipse.microprofile.lra.tck.participant.api;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.ws.rs.core.Response;
+import java.net.URI;
+
+
+@ApplicationScoped
+public class LRATypeTckBean {
+
+    public static final String REQUIRED_PATH = "/required";
+    public static final String REQUIRES_NEW_PATH = "/requires-new";
+    public static final String SUPPORTS_PATH = "/supports";
+    public static final String NOT_SUPPORTED_PATH = "/not-supported";
+    public static final String MANDATORY_PATH = "/mandatory";
+    public static final String NEVER_PATH = "/never";
+
+    public static final String REQUIRED_WITH_END_FALSE_PATH = "/end-required";
+    public static final String REQUIRES_NEW_WITH_END_FALSE_PATH = "/end-requires-new";
+    public static final String SUPPORTS_WITH_END_FALSE_PATH = "/end-supports";
+    public static final String NOT_SUPPORTED_WITH_END_FALSE_PATH = "/end-not-supported";
+    public static final String MANDATORY_WITH_END_FALSE_PATH = "/end-mandatory";
+    public static final String NEVER_WITH_END_FALSE_PATH = "/end-never";
+
+    // resource methods for each LRA.Type attribute reused in all LRA type tests
+
+    public Response requiredLRA(URI lraId) {
```

Comment:
All the methods in this class have the same body and thus can be replaced by a single method. (and even not required to have a method to represent a single simple statement)

Reply:
I made this decision based on thinking that if we are to change any of the methods you will need to update it in 3 places, but maybe this is an unnecessary optimization. I will revert back to returning response directly.